README.md

```diff
@@ -461,6 +461,37 @@ yarn lingui:compile
 
 [Lingui]: https://lingui.js.org/
 
+### Garbling message catalogs
+
+When internationalizing a piece of code, it can be difficult to
+tell if one missed any strings because of how our internationalization
+frameworks fall back to English when our strings haven't been translated
+yet.
+
+To compensate for this, we provide a way to "garble" message catalogs with
+nonsense localizations, which makes it easier to tell whether all our strings
+have been properly internationalized.  When enabled, it looks like this:
+
+> <img src="https://user-images.githubusercontent.com/124687/82383356-9b537380-99fb-11ea-8953-8c3c511591b2.png" alt="Screenshot of a page with question marks for all text">
+
+To activate the garbling, run:
```

Comment:
Maybe add something about the ideal dev workflow.
i.e. 
ONLY DO THIS IN INTERNATIONALIZATION PRs after the initial PR is in staging, or something

Should you garble before or after adding new strings? Does it matter?

Reply:
Hmm that's fairly contextual, but still worth documenting--I'm just not sure if it belongs in the readme since it's particular to our deployment workflow and our use of Crowdin.  Maybe we can link to the wiki from this section for an example of it in use in a workflow?